linux_os/guide/services/ntp/service_ntp_enabled/rule.yml

```diff
@@ -49,3 +49,10 @@ template:
     name: service_enabled
     vars:
         servicename: ntp
+
+{{% if prodtype in ["rhel8", "rhel9"] %}}
```

Comment:
`sle15` is also failing by the same reason. Can we also include the warning for `sle15`?

Reply:
Sure, I checked it on sle15 container and it makes sense.
Fixed in 1c077d69e620aecbb20cd18c42afdc02ecd5a555

Reply:
Thanks